packages/inferno/src/DOM/events/delegation.ts

```diff
@@ -101,6 +101,7 @@ function attachEventToDocument(name: string) {
     });
 
     dispatchEvents(event, event.target, isClick, name, eventData);
+    return true;
```

Comment:
This should probably be `return dispatchEvents(....)` and adding correct return values to dispatchEvent

Reply:
So change line 103 to 'return dispatchEvents(event, event.target, isClick, name, eventData);'?

Reply:
Or does it need more modifications to dispatchEvents?

Reply:
or maybe the correct value is: `return !event.cancelBubble;` so if event is not stopped it returns true and else false? What do you think?

Reply:
I honestly have no idea, I don't even understand when is the return value used and how.

Reply:
https://stackoverflow.com/questions/128923/whats-the-effect-of-adding-return-false-to-a-click-event-listener

Reply:
So that is a legacy alternative of preventDefault. If I'm not mistaken then to do this properly, all DOM elements in the hierarchy which handle events would have to follow this return value convention for the changes to dispatchEvents, to use it, to make sense. 
As that seems unreasonable to assume/require, I suggest to return undefined here, to preserve the existing behaviour.

Reply:
sure